docs/source/insights_upload_configuration.rst

```diff
@@ -0,0 +1,39 @@
+Installing the Insights Client
+------------------------------
+The Insights Client requires the insights-core which is why we coupled these repositories together inside of an insights directory. To work with the Insights Client, begin by cloning the repository::
+
+    mkdir insights && cd insights
+    git clone git@github.com:RedHatInsights/insights-core.git
+    git clone git@github.com:RedHatInsights/insights-client.git
+
+Setting Up a Virtual Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We recommend that you isolate your development work by using a virtual environment. Run the following command to set up a virtual environment::
+
+    pipenv shell
+
+Accessing the Insights Development Branches
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For QPC to access the Insights Client locally we need to checkout branches that are still in development.
+
+    cd insights-core
```

Comment:
I'm not sure why, but if you actually view this, it is not rendering correctly. It is lumping all commands onto the same line